src/app/widgets/inlineinput/inlineinput.component.html

```diff
@@ -1,6 +1,5 @@
 <div class="inlineinput-container"
-[class.read-only]="disabled"
-(clickOut)="closeClick()">
```

Comment:
Why you remove this from here and added it [here](https://github.com/fabric8-ui/fabric8-planner/pull/2714/files#diff-dbc24aee843bca8262d1c28fbaf591ccR142)?

Reply:
I removed outside click from here so that it doesn't apply on all the other inline-inputs like dynamic fields and so that it is only used for wi title inline-input